tpool/tpool.h

```diff
@@ -75,6 +89,7 @@ class task_group
   void set_max_tasks(unsigned int max_concurrent_tasks);
   void execute(task* t);
   void cancel_pending(task *t);
+  void get_stats(group_stats* stats);
```

Comment:
There should be a space before `*`, not after it.

Reply:
done